frontend/pages/index.html

```diff
@@ -7,13 +7,36 @@
 -->
 
 <head>
+    <title>Tetris - | Free Online Game | - </title>
+    <link rel="icon" href="logo22.png">
+    <link rel="stylesheet" href="../styles/index.css">
     <meta charset="utf-8">
+    <h1 class = "title"style = "text-align: center;"> <big> Tetris</big> </h1>
 </head>
 
-<body>
-    <div>
-        Hello Work Experience!
+
+<body style = "background-color:whitesmoke;">
+
+   <button class = "how-to">
+    <p>How to Play</p>
+   </button>
+
+    
+
+   

```

Comment:
I think you had the logo1 picture in here before - did you accidentally get rid of it somewhere?

Reply:
I removed it yes I thought it was outstanding too much.